products/src/main/java/dev/robert/products/presentation/HomeScreen.kt

```diff
@@ -67,24 +68,34 @@ import java.lang.Math.floor
 @RootNavGraph(start = true)
 fun HomeScreen(
     viewModel: ProductsViewModel = hiltViewModel(),
-    navigator: HomeScreenNavigator,
+   // navigator: HomeScreenNavigator,
```

Comment:
Well, this is the most weird part. Theoretically by removing the "//" this should work just fine.
I'm not sure if this is really a bug on your side or the Nav Library!